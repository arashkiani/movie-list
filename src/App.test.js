/*global shallow, mount, sinon */
import Component from './App'


describe('<App />', () => {
	it('Match Snapshot', () => {
		const wrapper = shallow(<Component />)
		expect(wrapper).toMatchSnapshot()
	})
})