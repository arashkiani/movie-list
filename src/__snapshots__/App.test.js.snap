// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> Match Snapshot 1`] = `
<ForwardRef>
  <ForwardRef>
    <ForwardRef>
      <a
        href="/"
        target="black"
      >
        ZMDB
      </a>
    </ForwardRef>
    <Connect(SelectRangeFilter) />
    <Connect(SelectGenresFilter) />
  </ForwardRef>
  <Connect(Collection) />
</ForwardRef>
`;
